widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.kt

```diff
@@ -40,9 +42,8 @@ import kotlin.properties.Delegates
  * startActivity(intent);
 </pre> *
  */
-class CallActivity : FadeTransitionActivity() {
+internal class CallActivity : FadeTransitionActivity() {
```

Comment:
👍 Finally! I tried to do this twice only to find there are still public methods/classes that require this to be public. Very cool that it is internal at last!